namespace cpp entity
namespace py ccentity.direction
namespace as3 com.dreamfire.sanguo.data.definition.entity

enum DirectionType {
  MIN = 0,
  EAST = 0,               // 东
  WEST,                   // 西
  SOUTH,                  // 南
  NORTH,                  // 北
  NORTHEAST,              // 东北
  NORTHWEST,              // 西北
  SOUTHEAST,              // 东南
  SOUTHWEST,              // 西南
  NONE,                   // NONE

  MAX,
}

